tests/test_clinical.py

```diff
@@ -705,8 +705,27 @@ def test__check_vital_status_consistentency_valid():
          "YEAR_1": [1, 4, "Unknown"],
          "FOO_3": [1, 3, "Unknown"]}
     )
-    genie_registry.clinical._check_vital_status_consistentency(
+    error = genie_registry.clinical._check_vital_status_consistentency(
         clinicaldf=testdf,
-        cols=['INT2', "YEAR1", "FOO_3"],
+        cols=['INT_2', "YEAR_1", "FOO_3"],
         missing_vals=["Unknown"]
-    )
\ No newline at end of file
+    )
+    assert error == ""
+
+
+def test__check_vital_status_consistentency_inconsistent():
```

Comment:
Suggestion for another check that should result in an error:

`clinicaldf = pd.DataFrame([[1, "Unknown"], ["Unknown", 3]], columns = cols)`

Reply:
We should also do something about the ">..." and "<...." values.  For instance if a YEAR_CONTACT column has ">89", the INT_CONTACT should have ">..." as well.

Reply:
agreed

Reply:
Ah.. I also realize that the unknown values should probably map to one another.  So if there is a "Unknown" it should technically map to an "Unknown" and not "Not Collected".  For instance, this could work:

```
clinicaldf = pd.DataFrame([[1, "Unknown"], [3, "Not Applicable"]], columns = cols)
```

Reply:
Did you mean...

`clinicaldf = pd.DataFrame([[1, 3], ["Unknown", "Not Collected"]], columns = cols)` ?

I think it would depend on the definition of the schema.  If all of the above are valid as representations of missing values, then having a mixture is fine even if the actual English word meanings are technically different.

Reply:
Ah yes, thats what I meant.  The problem is those string values mean different things.  So if `INT_DOD` is "Not Applicable", `YEAR_DEATH` should also be "Not Applicable".  It shouldn't be "Unknown..

The definition of those values are:

Not Released: Information cannot be released due to site IRB.
Not Collected: Information not collected (ie. PRIMARY_RACE isn't collected at some sites)
Not Applicable: Information cannot be filled out (ie. YEAR_DEATH can't be filled out if the patient is still alive)
Unknown: Patient/Sample information collected, but is unknown.

Reply:
I see.  Then I agree since they have specific meanings, the labels should be consistent.